manifest_staging/charts/gatekeeper/templates/upgrade-crds-hook.yaml

```diff
@@ -71,7 +71,7 @@ metadata:
     helm.sh/hook-weight: "1"
     helm.sh/hook-delete-policy: "hook-succeeded,before-hook-creation"
 spec:
-  backoffLimit: 0
+  backoffLimit: 3
```

Comment:
why not take the default? something less to configure?

Reply:
The default is 6 and retries are with exponential backoff _(I've mentioned this in the PR description as well)_ which could cause the job to hang for longer in case of actual failures. The goal with configuring it to `3` was so there are some retries but still we could fail fast. Let me know what you think!